internal/rootcoord/quota_center.go

```diff
@@ -83,13 +84,15 @@ type QuotaCenter struct {
 	dataCoord  types.DataCoord
 
 	// metrics
-	queryNodeMetrics map[UniqueID]*metricsinfo.QueryNodeQuotaMetrics
-	dataNodeMetrics  map[UniqueID]*metricsinfo.DataNodeQuotaMetrics
-	proxyMetrics     map[UniqueID]*metricsinfo.ProxyQuotaMetrics
-	dataCoordMetrics *metricsinfo.DataCoordQuotaMetrics
-
-	currentRates map[internalpb.RateType]Limit
-	quotaStates  map[milvuspb.QuotaState]commonpb.ErrorCode
+	queryNodeMetrics    map[UniqueID]*metricsinfo.QueryNodeQuotaMetrics
+	dataNodeMetrics     map[UniqueID]*metricsinfo.DataNodeQuotaMetrics
+	proxyMetrics        map[UniqueID]*metricsinfo.ProxyQuotaMetrics
+	dataCoordMetrics    *metricsinfo.DataCoordQuotaMetrics
+	readableCollections []int64
+	writableCollections []int64
+
+	currentRates map[int64]map[internalpb.RateType]Limit
```

Comment:
The two-layer map structure is very complex, define a new type to represent `map[internalpb.RateType]Limit`.